src/test/suite/didact.test.ts

```diff
@@ -37,11 +37,16 @@ const testMD2 = vscode.Uri.parse('vscode://redhat.vscode-didact?extension=demos/
 const testMD3 = vscode.Uri.parse('vscode://redhat.vscode-didact?extension=demos/markdown/validation-test.didact.md');
 const testExt = 'didact://?commandId=vscode.didact.extensionRequirementCheck&text=some-field-to-update$$redhat.vscode-didact';
 const testReq = 'didact://?commandId=vscode.didact.requirementCheck&text=os-requirements-status$$uname$$Linux&completion=Didact%20is%20running%20on%20a%20Linux%20machine.';
+const testReqWin = 'didact://?commandId=vscode.didact.requirementCheck&text=os-requirements-status$$echo%20test$$test&completion=Didact%20is%20running%20on%20a%20Windows%20machine.';
 const testReqCli = 'didact://?commandId=vscode.didact.cliCommandSuccessful&text=maven-cli-return-status$$uname&completion=Didact%20is%20running%20on%20a%20Linux%20machine.';
+const testReqCliWin = 'didact://?commandId=vscode.didact.cliCommandSuccessful&text=maven-cli-return-status$$echo%20test&completion=Didact%20is%20running%20on%20a%20Linux%20machine.';
```

Comment:
the message is "Didact%20is%20running%20on%20a%20Linux%20machine." but it is the testReqCliWin

Reply:
that is a good catch. I should just get rid of all the completion messages in the tests anyway